connectors/steps/redshift/query.py

```diff
@@ -0,0 +1,82 @@
+import json
+from connectors.serializers import get_serializer
+from connectors.serializers.base import BaseSerializer
+from connectors.utils.table_printer import print_truncated_table
+import redshift_connector
+import valohai
+import requests
+import csv
+import os
+
+
+# Redshift environment variables
+host = os.environ.get('REDSHIFT_HOST')
+db_name = os.environ.get('REDSHIFT_DATABASE')
+db_user = os.environ.get('REDSHIFT_USER')
+db_password = os.environ.get('REDSHIFT_PASSWORD')
+cluster_identifier = os.environ.get('REDSHIFT_CLUSTER_IDENTIFIER')
+region = os.environ.get('REDSHIFT_REGION')
+port = int(os.environ.get('REDSHIFT_PORT', 5439))
+use_iam = bool(os.environ.get('REDSHIFT_USE_IAM', False))
+
+# Other environment variables
+aws_metadata_ip = os.environ.get('AWS_METADATA_IP', '169.254.169.254')
+vh_worker_role = os.environ.get('VH_WORKER_ROLE', 'ValohaiWorkerRole')
+
+# Step parameters
+datum_alias = valohai.parameters('datum_alias').value
+output_path = valohai.parameters('output_name').value
```

Comment:
Should the name of this variable be `output_name`?

Reply:
Current implementation:
`output_path = valohai.outputs().path(output_path if output_path else 'results.csv')`

Which means you can set it as `hello/myoutput/jepa.csv` if you want.

Reply:
Yes. Ignore my earlier comment.